tst/tests/cast_equal.int

```diff
@@ -0,0 +1,5 @@
+; 69
```

Comment:
This comment doesn’t really belong here, but a test for the truncation case would be nice too. :eyes:

Reply:
... it's called `cast.int`

Reply:
Oh, nice. I only looked at the tests that are new in this branch